src/libraries/Common/src/Interop/Interop.Libraries.cs

```diff
@@ -5,6 +5,12 @@ internal static partial class Interop
 {
     internal static partial class Libraries
     {
+#if TARGET_WINDOWS
```

Comment:
Since the names are different, I think it would look better without ifdefs and in the platform-specific files. (ie delete this file).

Reply:
NP. I thought the idea is that files that are used by all platforms, even though with different names, go here.
Either way would work though.